### DISCUSSION TITLE
Default Formula for ispindel.

### DISCUSSION THREAD

**Discussion by andvenaa** [mp-se/gravitymon#138] [Ideas]
Hi,

I tried to use the GravityMon firmware on 2 of my iSpindel devices. And in iSpindel software there is a quick calibration. That is about the same as the **Calibrate device** that finds the 90°. I thougt that everything was good to go and dropped them in 2 different brews, to find out it didn't report gravity. :(

I don't use the iSpindel to get the "correct" values, only to see the trend.

So my solution was add another iSpindel with original firmware to get the readings.

So my 2 cents.
There should be some default values for the Formula so it "works" out of the box. And you could do the Gravity formula creation to get more accurate readings.

**Comment by mp-se** [comment]
That is a good suggestion for a new feature to at least get a gravity reading with a default installation. Most of the PCB's should give 25 deg in water if built correctly, so it should be possible to have some default values as a starting point.

I have a few builds so I will look into how their calibration points are and what the accuracy could be.

**Comment by mp-se** [comment]
I've added this feature and some updates to the UI in beta 2. You need to take action in order to add a default formula, but its just clicking a button in the UI.

**Reply by andvenaa** [reply]
Nice, will check it out after work. 👍
